src/css/main.css

```diff
@@ -1,285 +1,300 @@
 html {
-  font-size: 1em;
-  line-height: 1.5;
-  height: 100%;
-  box-sizing: border-box;
+    font-size: 1em;
+    line-height: 1.5;
+    height: 100%;
+    box-sizing: border-box;
 }
 
 body {
-  font-family: 'Quicksand', sans-serif !important;
+    font-family: 'Quicksand', sans-serif !important;
 }
 
 p {
-  margin-bottom: 0 !important;
+    margin-bottom: 0 !important;
 }
 
 a {
-  color: #0A4BED;
+    color: #0A4BED;
 }
 
 button:disabled {
-  cursor: not-allowed;
-  pointer-events: all !important;
+    cursor: not-allowed;
+    pointer-events: all !important;
 }
 
 .navbar {
-  border-bottom: 1px solid rgba(255, 255, 255, 0.3);
+    position: fixed;
```

Comment:
I think this breaks the smooth scrolling anchor. You may need to subtract the height of the nav bar in the JS click scroll handler

Reply:
It also makes the next section "poke out" at least on the splash screen. Is that okay with you? 
![image](https://user-images.githubusercontent.com/22859824/141029099-4d03161b-f63b-4e55-860f-36105453c395.png)

Reply:
ohhh I noticed the scrolling but wasn't sure where to fix it. I'll fix both of those!